core/arch/arm/plat-imx/imx-common.c

```diff
@@ -23,9 +23,9 @@ static uint32_t imx_digproc(void)
 
 		/* TODO: Handle SL here */
 #ifdef CFG_MX7
-		reg = read32(anatop_addr + OFFSET_DIGPROG_IMX7D);
+		reg = read32(anatop_addr + HW_ANADIG_DIGPROG_IMX7D);
```

Comment:
I would not to change this in upstream repo.

Reply:
if you don't mind i would like to keep them as is.